plugins/cassandra/cassandrawu.cpp

```diff
@@ -4299,7 +4299,7 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
         getFieldNames(searchMappings+3, names, tableName);  // mappings+3 means we don't return the key columns (xpath, upper(keyPrefix), upper(key))
         VStringBuffer selectQuery("select %s from %s where xpath=? and fieldPrefix=?", names.str()+1, tableName.str());
         if (threshold && *threshold)
-            selectQuery.appendf(" where fieldValue >= ?");
+            selectQuery.appendf(" and fieldValue >= ?");
```

Comment:
@richardkchapman - not really sure re. reasons/implications of this change - is it correct/ok?

Reply:
It looks like a bugfix to me (selectQuery already has a where clause, see 2 lines earlier)